backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/ObservationHandler.java

```diff
@@ -55,7 +57,7 @@ public ObservationHandler(Executor executor, FutureJdbi jdbi) {
                                 + "(select id, kv_value, value_type from keyvalue where kv_key =:name and entity_name IS NULL) k "
                                 + "where o.keyvaluemapping_id = k.id")
                         .bind("run_id", runId)
-                        .bind("entity_name", "\"ExperimentRunEntity\"")
+                        .bind("entity_name", "ExperimentRunEntity")
```

Comment:
Revert. This was done by Cory for MS SQL server compatibility

Reply:
resolved it while merged